functions/rust-mnist/www/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+node_modules
+dist
+dist-ssr
+*.local
+
+# Editor directories and files
+.vscode/*
```

Comment:
No development-environment-specific stuff in `.gitignore`. Users should set up their own global `.gitignore`.